core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.jelly

```diff
@@ -138,14 +144,14 @@ THE SOFTWARE.
                         </j:otherwise>
                     </j:choose>
                 </table>
-    
+                
```

Comment:
I'm not sure of the normal style. I personally am not a fan of trailing whitespace (and apparently the code wasn't using it before).

Reply:
Yeah I just tried to make it consistent